openreview/tools.py

```diff
@@ -37,8 +37,7 @@ def create_profile(client, email, first, last, middle = None, allow_duplicates =
     if not profile:
         response = client.get_tildeusername(first, last, middle)
         tilde_id = response['username']
-
-        if tilde_id.endswith(last + '1') or allow_duplicates:
+        if tilde_id.endswith('1') or allow_duplicates:
```

Comment:
can you describe the nature of the issue this function and special characters? what kind of error are you getting? is it an encoding issue?

this isn't great because it won't catch the 11'th duplicate (e.g. ~Michael_Spector11). This may seem ridiculous until you consider that there could easily be 11 "Wei Li"s in the CS community.

Reply:
I'm trying to reproduce the error.  I'll let you know when I do.

Reply:
also, I just realized that this commit is on a branch that is quite old - you may want to do a merge locally before you go further, so that you don't need to merge things after the fact

Reply:
For ~Antonio_Cubero_Fernández1 it produces this error:
 'ascii' codec can't decode byte 0xc3 in position 4: ordinal not in range(128)